api/config/rolepermission.php

```diff
@@ -0,0 +1,521 @@
+            'fr' => 'Afficher sa propre application'
+        ],
+        'view-team-submittedApplication' => [
+            'en' => 'View Team Submitted Application',
+            'fr' => 'Voir la demande soumise par l\'équipe'
+        ],
+        'view-team-applicantProfile' => [
+            'en' => 'View Team Applicant Profile',
+            'fr' => 'Voir le profil du candidat de l\'équipe'
+        ],
+        'create-own-draftApplication' => [
+            'en' => 'Create Own Draft Application',
+            'fr' => 'Créer son propre projet d\'application'
+        ],
+        'submit-own-application' => [
+            'en' => 'Submit Own Application',
+            'fr' => 'Soumettre sa propre demande'
+        ],
+        'update-team-applicationStatus' => [
+            'en' => 'Update Team Application Status',
+            'fr' => 'Mise à jour du statut de la demande de l\'équipe'
+        ],
+        'delete-own-draftApplication' => [
+            'en' => 'Delete Own Draft Application',
+            'fr' => 'Supprimer son propre projet de demande'
+        ],
+        'archive-own-submittedApplication' => [
+            'en' => 'Archive Own Submitted Application',
+            'fr' => 'Archiver sa propre demande'
+        ],
+        'view-any-applicantCount' => [
+            'en' => 'View Applicant Count',
+            'fr' => 'Voir le nombre de candidats'
+        ],
+
+        'create-any-searchRequest' => [
+            'en' => 'Create Any Search Request',
+            'fr' => 'Créer toute demande de recherche'
+        ],
+        'view-team-searchRequest' => [
+            'en' => 'Create Team Search Request',
+            'fr' => 'Créer une demande de recherche d\'équipe'
+        ],
+        'update-team-searchRequest' => [
+            'en' => 'Update Team Search Request',
+            'fr' => 'Mise à jour de la demande de recherche d\'équipe'
+        ],
+        'delete-team-searchRequest' => [
+            'en' => 'Delete Team Search Request',
+            'fr' => 'Supprimer une demande de recherche d\'équipe'
+        ],
+
+        'view-any-team' => [
+            'en' => 'View Any Team',
+            'fr' => 'Voir n\'importe quelle équipe'
+        ],
+        'view-any-teamUsersBasicInfo' => [
+            'en' => 'View Any Team Users Basic Info',
+            'fr' => 'Afficher les informations de base sur les utilisateurs de n\'importe quelle équipe'
+        ],
+        'view-team-teamUsersBasicInfo' => [
+            'en' => 'View Teams Team Users Basic Info',
+            'fr' => 'Afficher les informations de base sur les utilisateurs de l\'équipe'
+        ],
+        'create-any-team' => [
+            'en' => 'Create Any Team',
+            'fr' => 'Créer n\'importe quelle équipe'
+        ],
+        'update-any-team' => [
+            'en' => 'Update Any Team',
+            'fr' => 'Mise à jour de n\'importe quelle équipe'
+        ],
+        'update-team-team' => [
+            'en' => 'Update Team Team',
```

Comment:
Will folks using this know what this means? Is there another way to write this?

Reply:
I'm all for suggestions! Display name is a nullable value but I though these has some value so added them. Honestly, I was just writing the keys out as is because of the number of them I needed to convert.

I would love some input on better names for any of the permissions.

Reply:
Ahaha, I'm not sure what else to use as a key without making the general action-scope-resource nomenclature worse, but I'm open to suggestions. The display name can definitely be better. "Update my Teams"?